diff --git a/node_modules/chessground/config.d.ts b/node_modules/chessground/config.d.ts
index ed1c85c..7a360f1 100644
--- a/node_modules/chessground/config.d.ts
+++ b/node_modules/chessground/config.d.ts
@@ -29,7 +29,7 @@ export interface Config {
         dests?: cg.Dests;
         showDests?: boolean;
         events?: {
-            after?: (orig: cg.Key, dest: cg.Key, metadata: cg.MoveMetadata) => void;
+            after?: (orig: cg.Key, dest: cg.Key, metadata?: cg.MoveMetadata) => void;
             afterNewPiece?: (role: cg.Role, key: cg.Key, metadata: cg.MoveMetadata) => void;
         };
         rookCastle?: boolean;
